github-sync.sh

```diff
@@ -46,6 +46,13 @@ if [[ "$SYNC_TAGS" = true ]]; then
   git tag -d $(git tag -l) > /dev/null
   git fetch tmp_upstream --tags --quiet
   git push origin --tags --force
+elif [[ -n "$SYNC_TAGS" ]]; then
+  echo "Force syncing tags matching pattern: $SYNC_TAGS"
+  git tag -d $(git tag -l) > /dev/null
+  git fetch tmp_upstream --tags --quiet
+  for tag in $(git tag -l "$SYNC_TAGS"); do
+    git push origin "refs/tags/$tag" --force
+  done
```

Comment:
```suggestion
  git tag | grep "$SYNC_TAGS" | xargs --no-run-if-empty git push origin --force
```

We can optimize this loop like the above, should we?